src/router/routes_conf.ts

```diff
@@ -11,109 +12,127 @@ const routesConf: RoutesConf = {
   community: {
     path: '/community',
     children: {
+      fullPath: '/community',
       name: 'community',
     },
   },
   company_coordinator: {
     path: '/company-coordinator',
     children: {
+      fullPath: '/company-coordinator',
       name: 'company-coordinator',
     },
   },
   home: {
     path: '/',
     children: {
+      fullPath: '/',
       name: 'home',
     },
   },
   login: {
     path: '/login',
     children: {
+      fullPath: '/login',
       name: 'login',
     },
   },
   prizes: {
     path: '/prizes',
     children: {
+      fullPath: '/prizes',
       name: 'prizes',
     },
   },
   register: {
     path: '/register',
     children: {
+      fullPath: '/register',
       name: 'register',
     },
   },
-  'register-coordinator': {
+  register_coordinator: {
     path: '/register-coordinator',
     children: {
+      fullPath: '/register-coordinator',
       name: 'register-coordinator',
     },
   },
-  'register-challenge': {
+  register_challenge: {
     path: '/register-challenge',
     children: {
+      fullPath: '/register-challenge',
       name: 'register-challenge',
     },
   },
   results: {
     path: '/results',
     children: {
+      fullPath: '/results',
       name: 'results',
     },
   },
   results_detail: {
-    path: '/results/detail',
+    path: 'detail',
```

Comment:
Why some path has start with  `/`  and others not?

Reply:
@tmszi  with the way routes are nested, name starting with `/` indicates path root such as `domain/#/routes`
while paths without `/` are the nested parts of the url, such as `calendar` in `domain/#/routes/calendar`.

Reply:
It should be good if all URL path start with `/` or without `/`, but not mixed way.

Reply:
Not really. Root paths are required to start with `/` (you get an error message about that in the console).
However, if you start the sub-paths with `/`, the router interprets them as root paths and redirects e. g. to `domain/#/calendar` instead of `domain/#/routes/calendar`.

Reply:
Yes you are right, it is nested router path.